subworkflows/local/visualization_krona.nf

```diff
@@ -34,6 +42,25 @@ workflow VISUALIZATION_KRONA {
     ch_krona_text = ch_krona_text.mix( KRAKENTOOLS_KREPORT2KRONA.out.txt )
     ch_versions = ch_versions.mix( KRAKENTOOLS_KREPORT2KRONA.out.versions.first() )
 
+    /*
+        Combine Kaiju profiles with their databases
+    */
+    ch_input_for_kaiju2krona = ch_input_classifications.kaiju
+        .map{ [it[0]['db_name'], it[0], it[1]] }
+        .combine( databases.map{ [it[0]['db_name'], it[1]] }, by: 0 )
+        .multiMap{
+            it ->
+                profiles: [it[1], it[2]]
+                db: it[3]
+        }
```

Comment:
This isn't necessary (nor executing kaij2krona within this module), as (I forgot) we already have this in the profiling workflow:

https://github.com/nf-core/taxprofiler/blob/e66c2b9b87c99aa0935c89fae4ea92c1481b91e4/subworkflows/local/profiling.nf#L182

As we have to use the database for it (and I don't want to continue adding additional input channels). So this can be removed as line 60 here, and just have the mixing directly into the channel for Krona itself.

Reply:
I'm pretty sure it is necessary. `kaiju2table` and `kaiju2krona` output very different formats. `kaiju2table`'s output does not contain the taxonomy hierarchy, which `ktImportText` needs in a Krona text file. All my tests show blank Krona charts when importing directly from `kaiju2table`'s output, but if you can make this work without importing the database into `VISUALIZATION_KRONA`, I agree that would be better.

Reply:
Yes. It's good to go.